### PR TITLE
SEO Update and Mocopi link fix

### PR DESCRIPTION
**Pull request by ShineBrightMeow**

### CODE REVIEW

**Review by Eirenliel — APPROVED** [review]

File: theme/index.hbs [hunk 1]
```diff
@@ -6,10 +6,15 @@
...
         {{#if is_print }}
         <meta name="robots" content="noindex">
+        <meta name="googlebot" content="noindex">
         {{else}}
         <meta name="robots" content="index, follow">
+        <meta name="googlebot" content="index, follow">
         {{/if}}
 
+        <meta name="keywords" content="SlimeVR, Full, Body, Tracking, Tracker, Trackers, FBT, Virtual, Reality, VR, open, source, DIY, Documentation, Guide">
+        <link rel="canonical" href="https://docs.slimevr.dev/">
```
Comment (loucass003) [inline theme/index.hbs:16]: Cannonical should be defferent for each pages

**Review by loucass003 — CHANGES_REQUESTED** [review]

File: theme/index.hbs (hunk 1 above)
Comment (loucass003) [reply theme/index.hbs:16]: just remove it for now. We will see what the google search console says

**Review by loucass003 — APPROVED** [review]

File: theme/index.hbs (hunk 1 above)
Comment (ShineBrightMeow) [reply theme/index.hbs]: Canonical removed. Google does not use keyword meta like other search engines.

Comment (ShineBrightMeow) [reply theme/index.hbs]: But canonical should be fine since we are telling the search engine to index the main (home) page anyways.

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**